scripts/end_to_end_test/run_end_to_end_test.py

```diff
@@ -112,13 +112,16 @@ def setup_test(test_yaml, test_instance):
                                  expected=False, expect_type=int, default=10)
     pos_mode = yaml_extract(test_yaml, 'pos_mode', expected=False,
                             expect_type=bool, default=False)
+    num_lanes = yaml_extract(test_yaml, 'lanes', expected=False,
+                             expect_type=int, default=1)
 
     test_instance._number_of_nodes = number_of_nodes
     test_instance._node_load_directory = node_load_directory
     test_instance._node_connections = node_connections
     test_instance._nodes_are_mining = mining_nodes
     test_instance._max_test_time = max_test_time
     test_instance._pos_mode = pos_mode
+    test_instance._lanes = num_lanes
```

Comment:
I didn't follow it though in the E2E - does it def start with the right number of lanes?

Reply:
Yes